history/management/commands/run_bot.py

```diff
@@ -251,10 +266,19 @@ def predict(message,opponentname,gamename):
                     stats_for_sender['losses'] = stats_for_sender['losses'] + 1 
                     stats_for_sender['total'] = stats_for_sender['total'] + 1 
 
-            #send response
-            win_pct = round(stats_for_sender['wins'] * 1.0 / stats_for_sender['total'],2)*100
-            this_message = "{} total {} games played between {} vs {}.  {} is {}% likely to win next game".format(stats_for_sender['total'],gamename,sender,opponentname,sender,win_pct)
-            message.send(this_message)
+            win_pct = round(stats_for_sender['wins'] * 1.0 / stats_for_sender['total'],2)*100  
+            common_tag = Counter(list_tags).most_common()
+            if not common_tag:
+                this_message = "{} total {} games played between {} and {}. \n{} is {}% likely to win next game"\
+                               .format(stats_for_sender['total'],gamename,sender,opponentname,sender,win_pct)
+                message.send(this_message)
+            else:                
+                most_probable_tag = common_tag[0][0]
+                #send response
+                this_message = "{} total {} games played between {} and {}. \n{} is {}% likely to win next game by #{}"\
+                               .format(stats_for_sender['total'],gamename,sender,opponentname,sender,win_pct,most_probable_tag)
```

Comment:
since `list(Game.objects.filter(gamename=gamename,winner=sender,loser=opponentname))+list(Game.objects.filter(gamename=gamename,winner=opponentname,loser=sender)) 
` above on line 248, the most common tag _could_ be a game in which `sender` usually loses.  we should build a list of  common tags seperately for losses / wins.